### DISCUSSION TITLE
FireDBG: no precompiled binaries available for OS: Deepin20.9

### DISCUSSION THREAD

**Discussion by danielesk** [SeaQL/FireDBG.for.Rust#17] [Q&A]
system information
`Linux hw-PC 5.18.17-amd64-desktop-hwe  x86_64 GNU/Linux`

When I execute the command in the terminal ` curl https://raw.githubusercontent.com/seaql/firedbg.for.rust/main/install.sh -When using Sf | sh `, the following error occurs
**FireDBG: no precompiled binaries available for OS: Deepin20.9**

**Comment by billy1624** [comment]
Hey @danielesk, I'm not familiar with `Deepin`. Seems that it's a Linux distro forked from `Debian`. Perhaps you can manually installing FireDBG with prebuilt binaries of `Debian 12` / `Debian 10`.

Guide: https://github.com/SeaQL/FireDBG.for.Rust/blob/main/INSTALL.md#manual-installation

If this works, we can alter the `install.sh`, make it download `Debian` prebuilt binaries for `Deepin` OS. Just like what we did for Arch Linux, https://github.com/SeaQL/FireDBG.for.Rust/discussions/9

**Comment by danielesk** [comment]
@billy1624 
![Screenshot_select-area_20231230192306](https://github.com/SeaQL/FireDBG.for.Rust/assets/10652829/eab55f3a-7d23-443a-b2c5-2d1f947c5c8e)

I suggest optimizing the installation script. The optimization suggestions are as follows: detect files like /etc/debian_version under different systems, and if they exist, you can adjust them accordingly to trigger the installation.
